src/data/veilederbrukerknytning/useGetVeilederBrukerKnytning.tsx

```diff
@@ -24,8 +25,26 @@ export function useGetVeilederBrukerKnytning() {
   });
 }
 
+export const useVeilederHistorikk = () => {
+  const personident = useValgtPersonident();
+  const path = `${SYFOOVERSIKTSRV_PERSONTILDELING_ROOT}/historikk`;
+  const fetchHistorikk = () => get<[VeilederHistorikkDTO]>(path, personident);
+  return useQuery({
+    queryKey: veilederBrukerKnytningQueryKeys.historikk(personident),
+    queryFn: fetchHistorikk,
+    enabled: !!personident,
+  });
+};
+
 export interface VeilederBrukerKnytningDTO {
   personident: string;
   tildeltVeilederident?: string;
   tildeltEnhet?: string;
 }
+
+export interface VeilederHistorikkDTO {
```

Comment:
Kan dette være med på å gjøre hva denne representerer tydeligere? 😊
```suggestion
export interface VeilederTildelingHistorikkDTO {
```